torch/utils/bottleneck/__main__.py

```diff
@@ -223,9 +223,10 @@ def main():
     # if their execution times are very different.
     cuda_prof_exec_time = cpu_time_total(autograd_prof_cuda)
     cpu_prof_exec_time = cpu_time_total(autograd_prof_cpu)
-    pct_diff = cuda_prof_exec_time - cpu_prof_exec_time / cuda_prof_exec_time
-    if abs(pct_diff) > 0.05:
-        print_autograd_prof_summary(autograd_prof_cpu, 'CPU', autograd_prof_sortby, autograd_prof_topk)
+    if cuda_prof_exec_time > 1e-6:
+        pct_diff = cuda_prof_exec_time - cpu_prof_exec_time / cuda_prof_exec_time
```

Comment:
I'm wondering if this shouldn't have been
```python
(cuda_prof_exec_time - cpu_prof_exec_time) / cuda_prof_exec_time
```
cc @zou3519

Reply:
I think you are right.

Reply:
Yes, that looks reasonable.

I'm not sure what the type of 'cuda_prof_exec_time' and 'cpu_prof_exec_time' are -- if they're python ints we should not be using the '/' operator (or we should `from future import __division__` at the top of this file so it works out.

Reply:
cuda_prof_exec_time should be a float.